src/component-generators/vue/vue-component.ts

```diff
@@ -44,22 +46,23 @@ const createVueGenerator = (
     const cssChunks = result.chunks.filter((chunk) => chunk.meta.fileId === 'vuecss')
     const htmlChunks = result.chunks.filter((chunk) => chunk.meta.fileId === 'vuehtml')
 
-    const jsCode = chunksLinker.link(jsChunks)
-    const cssCode = chunksLinker.link(cssChunks)
-    const htmlCode = chunksLinker.link(htmlChunks)
+    const jsCode = removeLastEmptyLine(chunksLinker.link(jsChunks))
```

Comment:
does it make sense to move the `removeLastEmptyLine` call inside the `link` function? does it affect the React component generation?

Reply:
I don't know yet and didn't want to generate collateral effects.